flink-connectors/flink-connector-hive/src/main/java/org/apache/flink/table/catalog/hive/HiveCatalog.java

```diff
@@ -85,49 +86,39 @@ public void alterDatabase(String name, CatalogDatabase newDatabase, boolean igno
 
 	// ------ tables and views------
 
-	@Override
-	public void dropTable(ObjectPath tablePath, boolean ignoreIfNotExists)
-			throws TableNotExistException, CatalogException {
-		throw new UnsupportedOperationException();
-	}
-
-	@Override
-	public void renameTable(ObjectPath tablePath, String newTableName, boolean ignoreIfNotExists)
-			throws TableNotExistException, TableAlreadyExistException, CatalogException {
-		throw new UnsupportedOperationException();
-	}
-
 	@Override
 	public void createTable(ObjectPath tablePath, CatalogBaseTable table, boolean ignoreIfExists)
```

Comment:
I think we can keep the implementation only in HiveCatalogBase and define a new API called createHiveTable(). Each subcalss only needs to implement createHiveTable() (because they may create in different ways). I don't see why we need HiveCatalogUtil and related classes for this.

This might be cleaner.